cfgov/external_links/views.py

```diff
@@ -0,0 +1,76 @@
+from django.shortcuts import render
+from django.views.generic import View
+
+from external_links.forms import ExternalLinksForm
+
+from ask_cfpb.models.pages import AnswerPage
+from v1.models.base import CFGOVPage
+from v1.models.blog_page import BlogPage, LegacyBlogPage
+from v1.models.browse_filterable_page import BrowseFilterablePage
+from v1.models.browse_page import BrowsePage
+from v1.models.home_page import HomePage
+from v1.models.landing_page import LandingPage
+from v1.models.learn_page import (
+    AbstractFilterPage, DocumentDetailPage, EventPage, LearnPage
+)
+from v1.models.snippets import Contact, Resource, ReusableText
+from v1.models.sublanding_filterable_page import SublandingFilterablePage
+from v1.models.sublanding_page import SublandingPage
+
+
+class SearchView(View):
+    template_name = 'search.html'
+    # These are all v1 models that directly implement a visible part
+    # of the Wagtail page, specifically the content, sidefoot, or header
+    v1_page_models = [
```

Comment:
Oh I see - you mean for pages outside v1 that inherit from another page defined in `v1`?  Yeah, I think using `get_page_models` is a good way to make sure those get included.  I hadn't thought about that.   

Ideally this would be totally comprehensive, but I'm not sure for the first version of this I'll be adding the custom search indexes to all the non-v1 pages, since we'd need to audit those fields to see which ones need to be indexed.

Reply:
Updated to use `get_page_models` in a03716a43a00ecd2307d714193d90bc5dd136065